AfricanReligions/localisation/afr_general_localization.csv

```diff
@@ -26,7 +26,7 @@ GOD_TIURAKH;Tiurakh;;;;;;;;;;;;;x
 GOD_TAAHKARR;Taahkarr;;;;;;;;;;;;;x
 GOD_THE_EVIL_PANGOOLS;The evil pangools;;;;;;;;;;;;;x
 GOD_TANIT;Tanit;;;;;;;;;;;;;x
-GOD_ANTI;�nti;;;;;;;;;;;;;x
+GOD_ANTI;Änti;;;;;;;;;;;;;x
```

Comment:
It looks like you've converted these files into UTF-8.  You'll have to convert them back in Windows-1252/ANSI and then make sure your editor is set to that encoding.